packages/react/src/components/Cta/Cta.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { ArrowDown20, ArrowRight20, Launch20 } from '@carbon/icons-react';
+import { ButtonGroup } from '../../patterns/sub-patterns/ButtonGroup';
+import { CardLink } from '../../patterns/sub-patterns/CardLink';
+import { FeaturedLink } from '../../patterns/blocks/FeaturedLink';
+import { LinkWithIcon } from '../LinkWithIcon';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * CTA component
+ *
+ * @param {object} props props object
+ * @param {string} props.style style ( text | card | button | feature ).
+ * @param {object} props.type tyle ( jump | local | external ).
+ * @param {object} props.cta cta object which includes href,text, copy, card and image, button  properties.
+ * @returns {*} CTA Component
+ */
+const CTA = ({ style, type, ...cta }) =>
+  style === 'card' ? (
+    <CardLink {...cta} icon={iconSelector(type)} />
```

Comment:
I'm wondering if we should lock down the props for this given style. I'm not sure we want images, and other things in here. Only maybe a heading, and icon. Thoughts?

<img src="https://user-images.githubusercontent.com/3793636/73466367-65548b80-4347-11ea-8849-b0ae47a00e98.png" width="300px" />

Reply:
Yes, I would not pass all props in.

Reply:
@jeffchew good opportunity for those card decorator patterns we were talking about last week.